package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "akamai-edgeworkers-cli",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
I appreciate you updating the version number here. I had marked the tickets for this release as 1.6.0 but we can go with 1.5.2 as it's a bugfix update. I'll update the tickets in JIRA.